Lib/fontbakery/specifications/googlefonts.py

```diff
@@ -1071,12 +1071,10 @@ def com_google_fonts_check_082(metadata):
     yield SKIP, ("Found \"Multiple Designers\" at METADATA.pb, which"
                  " is OK, so we won't look for it at profiles.csv")
   else:
-    import urllib.request
-    import urllib.parse
-    import urllib.error
```

Comment:
Yeah, that was a `futurize` proposal. Sorry for applying this.

Reply:
no worries ;-)